pupil_src/shared_modules/video_overlay/plugins/eye_overlay.py

```diff
@@ -94,14 +96,20 @@ def _setup_eye(self, eye_id, prefilled_config):
         )
         return overlay
 
-    def _video_path_for_eye(self, eye_id):
+    def _video_path_for_eye(self, eye_id: int) -> str:
         rec_dir = self.g_pool.rec_dir
-        video_file_pattern = "eye{}.*".format(eye_id)
-        video_path_pattern = os.path.join(rec_dir, video_file_pattern)
-        try:
-            video_path_candidates = glob.iglob(video_path_pattern)
-            return next(video_path_candidates)
-        except StopIteration:
+
+        # Create eye video file filter for eye_id
+        file_filter = PupilRecording.FileFilter(rec_dir)
+        file_filter = file_filter.eye_id(eye_id)
+        file_filter = file_filter.videos()
+
+        # Get all eye videos for eye_id
+        eye_videos = list(file_filter)
```

Comment:
The intended usage for the FileFilter was more something like:
```python
recording = PupilRecording(rec_dir)
eye_videos = list(recording.files().videos().eye_id(eye_id))
```
with the `.files()` function there's no need to expose the FileFilter explicitly.
Not sure if this makes it clearer what's happening.

Reply:
@romanroibu Please adjust this accordingly in order to be consistent with other uses of the filefilter across the code base.